src/Uno.UWP/WinRTFeatureConfiguration.cs

```diff
@@ -64,5 +64,13 @@ public static class NetworkInformation
 		{
 			public static string ReachabilityHostname { get; set; } = "www.example.com";
 		}
+
+		public static class FileTypes
+		{
+			/// <summary>
+			/// Allows definition of custom file type MIME mappings.
+			/// </summary>
+			public static Dictionary<string, string> FileTypeToMimeMapping { get; } = new Dictionary<string, string>();
```

Comment:
Should this be documented ?

Reply:
I am planning to document when I do some full docs for all pickers (it will be used on other platforms too)